packages/web/package.json

```diff
@@ -17,7 +17,8 @@
     "apollo-utilities": "^1.3.3",
     "core-js": "3.6.0",
     "graphql": "^14.5.8",
-    "proptypes": "^1.1.0"
+    "proptypes": "^1.1.0",
+    "react-hook-form": "^4.5.5"
```

Comment:
This looks like it snuck in

Reply:
I do need that package for RedwoodForm but not for this PR, shouldn't have commited that one, whoops!